products/kubic/main.pm

```diff
@@ -67,10 +79,15 @@ if (get_var 'STACK_ROLE') {
 else {
     loadtests 'boot_from_dvd';
     if (get_var 'SYSTEM_ROLE') {
-        loadtests 'installation';
-        loadtests 'boot_from_disk';
-        loadtests 'feature' if (check_var 'EXTRA', 'FEATURES');
-        loadtest 'shutdown/shutdown';
+        if (check_var('HDDSIZEGB', '10')) {
+            loadtests 'boo#1099762';
+        }
+        else {
+            loadtests 'installation';
```

Comment:
As we have that loadtests wrapper, why not to use it too?

Reply:
This is what I am using. But, beware that to check that the bug is not present, I need that the test fails on the overview, and to have a green test, I need to omit following test modules.

In my opinion the **loadtests** wrapper would only allow to reduce duplicate code, if there weren't any schedule variant.
In this case, my fix is new schedule variant. And splitting the existing schedule groups (installation = preinstallation + installation + postinstallation) would cause an artificial less readable code.

Do you have any better suggestion? Or should I remove the wrapper?
(CC: @kravciak, @sysrich)